src/library_dylink.js

```diff
@@ -364,7 +364,8 @@ var LibraryDylink = {
       return zeroMemory(_malloc(size), size);
     }
     var ret = ___heap_base;
-    var end = (ret + size + 15) & -16;
+    // Keep __heap_base stack aligned.
```

Comment:
Why does this comment refer to `__heap_base`? Don't we use `getMemory` for more things?

Reply:
The `end` value here is used solely to update `__heap_base`.     Its doesn't have any other purpose.   This code path is for when the runtime is not yet initialized.

Reply:
I see, thanks. We used to use it elsewhere, which confused me.